edgelet/version.txt

```diff
@@ -1 +1 @@
-1.1.13
\ No newline at end of file
+1.1.14
```

Comment:
We don't need to update the `edgelet` version as we do NOT update the edgelet component.